baselines/sharepoint.md

```diff
@@ -1,16 +1,19 @@
+**`TLP:CLEAR`**
 # CISA M365 Security Configuration Baseline for SharePoint Online and OneDrive
 
-SharePoint Online is a web-based collaboration and document management platform. It is primarily used to collaborate on documents and communicate information in project teams. OneDrive is a cloud-based file storage system primarily used to store a user's personal files but it can also be used to share documents with others. This Secure Configuration Baseline (SCB) provides specific policies to strengthen the security of both of these services.
+SharePoint Online is a web-based collaboration and document management platform. It is primarily used to collaborate on documents and communicate information in projects. OneDrive is a cloud-based file storage system primarily used to store a user's personal files, but it can also be used to share documents with others. This secure configuration baseline (SCB) provides specific policies to strengthen the security of both services.
```

Comment:
Microsoft 365 (M365) SharePoint Online is a web-based collaboration and document management platform. It is primarily used to collaborate on documents and communicate information in projects. M365 OneDrive is a cloud-based file storage system primarily used to store a user's personal files, but it can also be used to share documents with others. This secure configuration baseline (SCB) provides specific policies to strengthen the security of both services. 

Changes to front-matter to standardize across SCBs.

Reply:
Change was adding "Microsoft 365 (M365)" to the first sentence and add "M365" to the start of the 3rd sentence which starts with OneDrive.

Reply:
I copied and pasted the entire paragraph you provided so hopefully it should be good.